lms/static/sass/shared-v2/_components.scss

```diff
@@ -0,0 +1,49 @@
+// TODO: tabs should be added to the Pattern Library
+.tabs {
+    @include clearfix();
+    @extend %reset-lists;
```

Comment:
Are we still following the Talbotsford property ordering method? If so, the extend should come first.